lib/coursemology/evaluator/client.rb

```diff
@@ -9,7 +9,11 @@ def self.initialize(host, api_user_email, api_token)
   end
 
   # @param [Boolean] one_shot If the client should only fire one request.
-  def initialize(one_shot = false)
+  # @param [Float] poll_time Sleep time between checks for task allocation.
+  # @param [String] poll_interval Units for poll_time.
+  def initialize(one_shot = false, poll_time = 10, poll_interval = 'seconds')
```

Comment:
I prefer a string with the unit attached and parsing be done. "10s" or "00:10" or the like. There should be ActiveSupport helpers for that.